README.md

```diff
@@ -45,6 +45,16 @@ the 5th version built against Kubernetes 1.20 APIs.
 This controller is developed and tested against Rackspace OpenStack Public Cloud using Go 1.16.
 Pull requests are welcome.
 
+## How to run locally. 
+
+1. In order to test your changes you will need a cloudspace. If you don't have a cloudspace then create one.
+2. Obtain the kubeconfig for your cloudspace. We will later use this to create a Kubernetes resource for our testing.
+3. Identity namespace for your KDU on staging cluster. 
+4. Identify the cloud-provider deployment which is responsible for your cloudspace in the namespace obtained from step-3.
+5. Scale down the cloud-provider deployment for your cloudspace to zero.
```

Comment:
Drop these. These are specific instructions to a project. It should just call out to stop any cloud-provider deployment running in your kuberenetes cluster.